main/windows/main.js

```diff
@@ -22,14 +22,24 @@ var config = require('../../config')
 var log = require('../log')
 var menu = require('../menu')
 var tray = require('../tray')
+var State = require('../../renderer/lib/state')
 
 var HEADER_HEIGHT = 37
 var TORRENT_HEIGHT = 100
+var state
 
 function init () {
   if (main.win) {
     return main.win.show()
   }
+
+  State.load(createWindow)
+}
+
+function createWindow (err, _state) {
+  if (err) return onError(err)
```

Comment:
Instead of returning on state.load error, we should just load the default window settings.